mt_dnn/model.py

```diff
@@ -36,10 +36,18 @@ def __init__(self, opt, state_dict=None, num_train_step=-1):
         self.total_param = sum([p.nelement() for p in self.network.parameters() if p.requires_grad])
 
         no_decay = ['bias', 'gamma', 'beta', 'LayerNorm.bias', 'LayerNorm.weight']
-        optimizer_parameters = [
-            {'params': [p for n, p in self.network.named_parameters() if n not in no_decay], 'weight_decay_rate': 0.01},
```

Comment:
something like?
 {'params': [p for n, p in param_optimizer if not any(nd in n for nd in no_decay)], 'weight_decay': 0.01}

Reply:
sure, I will make it that way.